RELEASE-NOTES.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [v1.16]
```

Comment:
In previous releases we have started with `.0` so we probably ought to do that here too.